joblib/test/test_memory.py

```diff
@@ -514,6 +514,41 @@ def test_call_and_shelve_argument_hash(tmpdir):
         in str(w[-1].message)
 
 
+def test_call_and_shelve_performance(tmpdir):
```

Comment:
`_performance` is really too generic a name. Please rename to "test_call_and_shelve_lazily_load_stored_result".